sql/hive/src/test/resources/regression-test-SPARK-8489/Main.scala

```diff
@@ -34,7 +34,7 @@ object Main {
     // scalastyle:off println
     println("Running regression test for SPARK-8489.")
     val sc = new SparkContext("local", "testing")
-    val sparkSession = SparkSession.withHiveSupport(sc)
+    val sparkSession = SparkSession.builder.enableHiveSupport().getOrCreate()
```

Comment:
yeah we need to rebuild the jar. There's already a TODO on L31. Right now the corresponding test in `HiveSparkSubmitSuite` is ignored.